.github/workflows/xlog-cron-compare.yml

```diff
@@ -27,21 +27,18 @@ jobs:
         working-directory: transition-scripts
 
       - name: Rocket.Chat Notification
-        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@master
         if: failure()
+        uses: fjogeleit/http-request-action@v1
         with:
-          type: ${{ job.status }}
-          job_name: '*One of the production xlogs failed to synch*'
-          mention: 'jsharman @nithinshekar.kuruba @Marco'
-          mention_if: 'failure'
-          channel: '#sso-alerts'
           url: ${{ secrets.SSO_ALERTS }}
-          token: ${{ secrets.ROCKETCHAT_TOKEN }}
+          method: "POST"
+          customHeaders: '{"Content-Type": "application/json"}'
+          data: '{"text": "@jsharman @nithinshekar.kuruba @jlanglois @Marco *One of the production xlogs failed to synch*", "attachments": [{"color": "#F5455C","title": "Details", "title_link": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}", "text": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}", "fields": [{"title": "Ref", "value": "${{ github.ref }}", "short": false}, {"title": "Workflow", "value": "${{ github.workflow }}", "short": false}]}]}'
   keepalive:
     name: Keepalive Workflow
     runs-on: ubuntu-latest
     permissions:
       actions: write
     steps:
       - uses: actions/checkout@v4
-      - uses: gautamkrishnar/keepalive-workflow@6112edb887975f902d74a6a66b688aed71932a70
+      - uses: gautamkrishnar/keepalive-workflow@v2
```

Comment:
For this action I had it pegged to the commit hash because it is a smaller community action with actions permissions, I wanted to keep it as an immutable release

Reply:
Thank you Jon, I have reverted it.